README.md

```diff
@@ -247,112 +201,114 @@ export GITHUB_AUTH_TOKEN=<your access token>
-A confidence of 0 should indicate the check was unable to achieve any real signal, and the result
-should be ignored.
-A confidence of 10 indicates the check is completely sure of the result.
-
-Many of the checks are based on heuristics, contributions are welcome to improve the detection!
-
-### Running specific checks
-
-To use a particular check(s), add the `--checks` argument with a list of check
-names.
-
-For example, `--checks=CI-Tests,Code-Review`.
+Each check returns a **Pass / Fail** decision, as well as a confidence score
+between **0 and 10**. A confidence of 0 should indicate the check was unable to
+achieve any real signal, and the result should be ignored. A confidence of 10
+indicates the check is completely sure of the result.
 
 ### Formatting Results
 
-There are three formats currently: `default`, `json`, and `csv`. Others may be added in the future.
+There are three formats currently: `default`, `json`, and `csv`. Others may be
+added in the future.
 
 These may be specified with the `--format` flag.
 
-## Requirements
-
-- The scorecard must only be composed of automate-able, objective data. For example, a project having 10 contributors doesn’t necessarily mean it’s more secure than a project with say 50 contributors. But, having two maintainers might be preferable to only having one - the larger bus factor and ability to provide code reviews is objectively better.
-- The scorecard criteria can be as specific as possible and not limited general recommendations. For example, for Go, we can recommend/require specific linters and analyzers to be run on the codebase.
-- The scorecard can be populated for any open source project without any work or interaction from maintainers.
-- Maintainers must be provided with a mechanism to correct any automated scorecard findings they feel were made in error, provide "hints" for anything we can't detect automatically, and even dispute the applicability of a given scorecard finding for that repository.
-- Any criteria in the scorecard must be actionable. It should be possible, with help, for any project to "check all the boxes".
-- Any solution to compile a scorecard should be usable by the greater open source community to monitor upstream security.
+## Public Data
 
-## Troubleshooting
+If you're only interested in seeing a list of projects with their Scorecard
+check results, we publish these results in a
+[BigQuery public dataset](https://cloud.google.com/bigquery/public-data).
 
-- ### Bugs and Feature Requests:
-  If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://github.com/ossf/scorecard/issues)
-Before you file an issue, please search existing issues to see if your issue is already covered.
+This data is available in the public BigQuery dataset
+`openssf:scorecardcron.scorecard`. The latest results are available at
+`openssf:scorecardcron.scorecard_latest`.
 
```

Comment:
Good to give example command to query results from data or lets say how to export them to csv or any other format.

Reply:
Done.